gef-ui/server/controllers/exporters-address/index.js

```diff
@@ -25,9 +25,10 @@ const validateExportersAddress = async (req, res) => {
     const { params, body } = req;
     const { correspondence, postcode } = body;
     const { applicationId } = params;
-    const { exporterId } = await api.getApplication(applicationId);
-    const { details } = await api.getExporter(exporterId);
-    const { registeredAddress } = details;
+
+    const { exporter } = await api.getApplication(applicationId);
+    const { companyName = '' } = exporter;
```

Comment:
Is this statement required?

Reply:
Yes, it's used in the template - if it's undefined it will throw error during this destructuring - so defaulting it to an empty string.